configs/default/FOXE-FOXEERH743.config

```diff
@@ -0,0 +1,153 @@
+# Betaflight / STM32H743 (SH74) 4.3.0 Jan 16 2022 
+
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6000
+#define USE_BARO_DPS310
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name FOXEERH743
+manufacturer_id FOXE
+
+# resources
+resource BEEPER 1 D02
+resource MOTOR 1 B04
+resource MOTOR 2 B05
+resource MOTOR 3 B00
+resource MOTOR 4 B01
+resource MOTOR 5 D12
+resource MOTOR 6 D13
+resource MOTOR 7 C08
+resource MOTOR 8 C09
+resource SERVO 1 E05
+resource SERVO 2 E06
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 7 E08
+resource SERIAL_TX 8 E01
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 7 E07
+resource SERIAL_RX 8 E00
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 C12
+resource CAMERA_CONTROL 1 B03
+resource ADC_BATT 1 C03
+resource ADC_RSSI 1 C05
+resource ADC_CURR 1 C02
+resource FLASH_CS 1 A15
+resource OSD_CS 1 A04
+resource GYRO_EXTI 1 D00
+resource GYRO_CS 1 B12
+
+# timer
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer D12 AF2
+# pin D12: TIM4 CH1 (AF2)
+timer D13 AF2
+# pin D13: TIM4 CH2 (AF2)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer E05 AF4
+# pin E05: TIM15 CH1 (AF4)
+timer E06 AF4
+# pin E06: TIM15 CH2 (AF4)
+
+# dma
+dma ADC 1 8
+# ADC 1: DMA1 Stream 1 Request 9
+dma ADC 2 9
+# ADC 2: DMA2 Stream 1 Request 10
+dma ADC 3 10
+# ADC 3: DMA2 Stream 2 Request 115
+
+dma TIMUP 1 0
+# TIMUP 1: DMA1 Stream 0 Request 15
+dma TIMUP 3 0
+# TIMUP 3: DMA1 Stream 0 Request 27
+dma TIMUP 4 0
+# TIMUP 4: DMA1 Stream 0 Request 32
+dma TIMUP 5 0
+# TIMUP 5: DMA1 Stream 0 Request 59
+dma TIMUP 8 0
+# TIMUP 8: DMA1 Stream 0 Request 51
+
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+dma pin D12 6
+# pin D12: DMA1 Stream 6 Request 29
+dma pin D13 7
+# pin D13: DMA1 Stream 7 Request 30
+dma pin C08 2
+# pin C08: DMA1 Stream 2 Request 49
+dma pin C09 3
+# pin C09: DMA1 Stream 3 Request 50
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature LED_STRIP
```

Comment:
this is now a user define so remove as it can be not included in the build

Reply:
LED_STRIP

Reply:
got it

Reply:
nice